src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -462,6 +463,24 @@ protected function getContextShopId()
      */
     protected function getErrorMessageForException(Exception $e, array $messages)
     {
+        if ($e instanceof ModuleEloquentException) {
+            return $e->getMessage();
+        }
+
+        $whitelist = [];
+        $this->get('prestashop.core.hook.dispatcher')->dispatchWithParameters('actionGetEloquentExceptionsFromModule', [
```

Comment:
Hook name can be improved. Once we have settled it, I'll register in SQL upgrade and hook.xml file.

Reply:
in the same spirit as for the exception name: `actionGetDisplayedExceptionsFromModule` `actionGetAlertExceptionsFromModule`